src/pages/2024/barcelona/travel.astro

```diff
@@ -1,67 +1,242 @@
+          experience for all.
+        </p>`}
+          />
+
+          <Accordion
+            client:load
+            title="Visas"
+            content={`<p>
+          EU citizens do not need a visa to visit Barcelona. Citizens of non-EU
+          countries may require a visa and must possess a passport valid for at
+          least three months. Please check the entry requirements for Spain to
+          review visa rules for your own situation.
+        </p>
+        <p>
+          If you need to apply for a visa to enter Spain we can issue you with a
+          personalized invitation letter to assist your application. If this
+          pertains to you please contact us by email at <a
+            href="mailto:help.summit@nextflow.io"
+            >help.summit@nextflow.io
           </a>
+        </p>`}
+          />
+          <Accordion
+            client:load
+            title="From Barcelona El Prat Airport"
+            content={`      <p>
+          The Aerobus is an express shuttle bus service between Barcelona
+          Airport and Barcelona city center. There are Aerobuses to and from
+          Terminal 1 and Terminal 2. The TMB Airport Bus Number 46 and Night Bus
+          N17 and N16 are also available for transfers between the airport and
+          the city center. TMB buses are cheaper but are also slower, less
+          frequent, and provide less luggage space than the Aerobuses.
+        </p>`}
+          />
+          <Accordion
+            client:load
+            title="City transport"
+            content={`        <p>
+          Mobility in Barcelona is easy as a result of its extensive metro,
+          tram, bus, and the city and suburban rail services. The city also has
+          a public bike hire scheme and more than 200 km of bike lanes, as well
+          as a taxi fleet that includes environmentally friendly options.
+        </p>`}
+          />
+          <Accordion
+            client:load
+            title="Code of Conduct"
+            content={`        
+            <ul class="list">
+          <li>Be nice and respectful to others</li>
+          <li>
+            Collaborate to create a safe and supportive environment for all
+            participants
+          </li>
+          <li>
+            Refrain from harassing speech, content, and other unwelcome behavior
+          </li>
+          <li>Be open to alternate points of view</li>
+          <li>Respect the privacy of others</li>
+          <li>
+            Comply with the instructions of the event staff and alert us if
+            something is wrong
+          </li>
+        </ul>
+        <p class="mt-6">You can check the full version of the Code of Conduct clicking   <a
+            target="_blank"
+            href="https://nf-co.re/code_of_conduct"
+          >here. </a></p>`}
+          />
         </Box>
       </div>
+      <Accommodation location="barcelona" client:load />
```

Comment:
This component has a `container` class, so it ends up with double padding. Need to break it out of the top-level `container` to avoid that and get consistent widths.

```suggestion
    </div>
    <Accommodation location="barcelona" client:load />
    <div class="container">
```